mlir/lib/Dialect/Rock/IR/RockDialect.cpp

```diff
@@ -532,6 +532,10 @@ GemmSize GemmSize::fromConvolution(ConvOpType type,
   return GemmSize(gemmGSize, gemmMSize, gemmKSize, gemmNSize);
 }
 
+static bool isFloat8Type(Type type) {
```

Comment:
No specific predicate for float-type-width.  This one is used in two places that could accept any of the four possibilities.